FIO/local-volumes/README.md

```diff
@@ -95,3 +97,95 @@ injected into the Job.
 ```bash
 ~$ kubectl logs $POD
 ```
+
+
+## DBench FIO Tests
+
+### Local Volume with no replicas
+
+The script `dbench-job-generator-local-volume.sh` provisions a StorageOS Volume
+and a Pod on the same node and then will execute a series of FIOs (15s per FIO
+test) on the provisioned volume to measure StorageOS performance. It requires
+the StorageOS CLI running as a pod in the cluster.
```

Comment:
I think it would be nice to add a link to the docs on how to get CLI running as a pod here.

Reply:
Agreed - or possibly an idempotent script which does it for us - see comment elsewhere.